kahuna/public/js/edits/metadataDiff.js

```diff
@@ -0,0 +1,26 @@
+export const getMetadataDiff = (image, metadata) => {
+  const diff = {};
+  const originalMetadata = image.data.originalMetadata;
+  // jscs has a maximumLineLength of 100 characters, hence the line break
+  var keys = new Set(
+    Object.keys(metadata).concat(Object.keys(originalMetadata))
+  );
+
+  keys.forEach((key) => {
+    if (originalMetadata[key] === metadata[key]) {
+      return;
+    }
+
+    //This only works with string fields and does not support arrays
+    if (Array.isArray(metadata[key]) || Array.isArray()) {
```

Comment:
`Array.isArray()` will always return false – what's the purpose of this second condition?

Reply:
That's supposed to be checking the other potential value for an array